src/utils.js

```diff
@@ -98,7 +105,7 @@ export const EventUtil = {
 export const NewsUtil = {
 	fetchBookingNews: () => {
 		const url =
-			"/ocs/v2.php/apps/activity/api/v2/activity/appointment?format=json";
+			"/ocs/v2.php/apps/activity/api/v2/activity/appointment?format=json&limit=5";
```

Comment:
Looking at it, 5 might be too low. I think we should go with the 10 instead for both.